docs/docs/plugins.md

```diff
@@ -379,6 +379,223 @@ plugin {
+    if (id) this.id = id;
+    this.dataTest = dataTest;
+    ...
+  }
+}
+```
+
+Also update the type definition to reflect this new property:
+
+**File:** `src/extensible-areas/floating-window/types.ts`
+
+```ts
+export interface FloatingWindowProps {
+  id: string;
+  dataTest: string;
+  ...
+}
+```
+
+---
+
+#### 5. Publish the changes to the projects needed
+
+To verify your change, you’ll need to test it in a sample plugin and in the BigBlueButton client.
+
+For new features, it’s best to create a new sample plugin. However, in this example, we'll use the existing `sample-floating-window-plugin`.
+
+Follow these steps to build and publish the SDK to both the sample plugin and `bbb-html5`:
+
+```bash
+# From the SDK root
+npm install
+npm run build
+
+# Ensure dependencies are installed in both target projects
+cd ~/path/to/bigbluebutton-html5
+npm install
+cd -
+
+cd samples/sample-floating-window-plugin
+npm install
+cd -
+
+# Publish SDK changes to both the BBB client and the sample plugin
+./scripts/publish-to-project-folder.sh ~/path/to/bbb-html5
+./scripts/publish-to-project-folder.sh samples/sample-floating-window-plugin
+```
+
+This ensures the changes in the SDK are applied to both the BigBlueButton client and the plugin sample.
+
+> **Optional:** To publish the SDK to all sample plugins at once, use:
+> ```bash
+> ./scripts/publish-to-samples.sh
+> ```
+
+---
+
+#### 6. Update the BigBlueButton Client (`bbb-html5`)
+
+Now make the necessary changes to the `bigbluebutton/bigbluebutton` repository in the `bigbluebutton-html5` directory. For our example, update the following files to pass and use the new `dataTest` prop:
+
+**File:** `imports/ui/components/floating-window/component.tsx`
+
+```tsx
+const renderComponent = (
+  elementRef: React.MutableRefObject<null>,
+  ...
+  dataTest: string,
+) => (
+  <Styled.FloatingWindowContent
+    ref={elementRef}
+    id={key}
+    data-test={dataTest}
+    ...
+  />
+);
+```
+
+Update the container (`imports/ui/components/floating-window/container.tsx`) logic to pass `dataTest` down to the `renderComponent`.
+
+---
+
+#### 7. Update the Sample Plugin
+
+After the SDK is built and published, you can update the sample plugin to use the new `dataTest` property accordingly. And test it (using the [development mode](#running-the-plugin-from-source), for example) against the BigBlueButton's client (`bigbluebutton-html5`).
+
+---
+
+#### 8. Submitting Pull Requests
+
+Once all changes are complete and tested:
+
+- Commit the changes in the `bigbluebutton-html-plugin-sdk` (all needed to run the feature/fix and the changes made to the sample);
+- In the `bigbluebutton-html5` client **do not reference a version number** for the SDK in `package.json` yet. Instead, use the commit hash from your SDK changes
+
+Example:
+
+```json
+"dependencies": {
+  ...
+  "bigbluebutton-html-plugin-sdk": "https://codeload.github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/tar.gz/<commit-hash-id>"
```

Comment:
FYI: In my tests I used a different approach referencing the PR instead of the commit:
`https://codeload.github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/tar.gz/refs/pull/181/head`

This way looks a bit simpler in my opinion.